x/evm/keeper/utils.go

```diff
@@ -0,0 +1,103 @@
+package keeper
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	authante "github.com/cosmos/cosmos-sdk/x/auth/ante"
+	"github.com/palantir/stacktrace"
+
+	evmtypes "github.com/tharsis/ethermint/x/evm/types"
+
+	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+	"github.com/ethereum/go-ethereum/core"
+	ethtypes "github.com/ethereum/go-ethereum/core/types"
+)
+
+// AccountKeeper defines an expected keeper interface for the auth module's AccountKeeper
+type AccountKeeper interface {
+	authante.AccountKeeper
+	NewAccountWithAddress(ctx sdk.Context, addr sdk.AccAddress) authtypes.AccountI
+	GetSequence(sdk.Context, sdk.AccAddress) (uint64, error)
+}
+
+// BankKeeper defines an expected keeper interface for the bank module's Keeper
+type BankKeeper interface {
+	authtypes.BankKeeper
+	GetBalance(ctx sdk.Context, addr sdk.AccAddress, denom string) sdk.Coin
+}
```

Comment:
these are already defined in `types/interface.go` we can merge the 2 probably and use those

Reply:
AccountKeeper's `GetParams` was the only method missing in the `interfaces.go` file. I added it and now every file is now using the same interfaces.